Dockerfile

```diff
@@ -1,26 +1,28 @@
-FROM summerwind/actions-runner-dind:ubuntu-20.04 as prod
+FROM ghcr.io/actions/actions-runner:latest AS prod
```

Comment:
I would strongly prefer to fix the version. This `:latest` has bitten me too many times. This can break things in a such a miraculous ways :smile: that you will spend a day fixing it. We can keep the bugfix version open if you prefer it